features/import_export/export_command.feature

```diff
@@ -0,0 +1,38 @@
+@importexport
+Feature: Export command
+    In order to export data from my store
+    As a store owner
+    I want to be able to run jobs based on defined export profiles
+
+    Background:
+        Given there are following export profiles configured:
+          | name               | description | code        | reader   | reader_configuration                  | writer     | writer_configuration                         |
+          | UsersExportProfile | Lorem ipsum | user_export | user_orm | batch_size:10,date_format:Y-m-d H:i:s | csv_writer | Delimiter:;,Enclosure:",File:/tmp/output.csv |
+        And there is default currency configured
+        And there is default channel configured
+        And there are following users:
+          | email          | enabled  | created_at          |
+          | beth@foo.com   | yes      | 2010-01-02 12:00:00 |
+          | martha@foo.com | yes      | 2010-01-02 13:00:00 |
+          | rick@foo.com   | yes      | 2010-01-03 12:00:00 |
+        And I am logged in as administrator
+
+    @scenarioWithFile
+    Scenario: Running export command
+         When I run "sylius:export user_export" command in less then "300" seconds
+         Then the command should finish successfully
+          And I should see "Command executed successfully!" in terminal
+          And I should find "/tmp/output.csv" file
+          And this file should contains 4 rows
+          And the file data should be valid
+          And I should find 1 "completed" job for this "export" profile in database
```

Comment:
Not really. As far as I remember this is needed to make is available for both export and import. Another thing is that this step allow to check different types of job, like "completed" in this example.